api/go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/photoview/photoview/api
 
-go 1.22.0
+go 1.22.5
```

Comment:
Note: `go mod tidy` updates this version and adds `toolchain` version, based on all dependencies. It's good to let `go mod tidy` to maintain it.